src/views/layout.ejs

```diff
@@ -1,18 +1,30 @@
 <html>
-  <head>
-    <title>GirlsCodeMK Library app</title>
-  </head>
 
-  <body>
-    <a href="/">Home</a>
+<head>
+  <title>GirlsCodeMK Library app</title>
+  <link rel='stylesheet' type='text/css' href='/css/styles.css'>
+</head>
+
+<body>
+  <div class="container">
+    <a href="/"><img src="/img/logo.png" alt="library-app-logo">
+    </a>
+
+
     <% if (messages.alert) { %>
-      <div class='alert'><%- messages.alert %></div>
+    <div class='alert'>
```

Comment:
I can imagine VSCode is being a nuisance here because of the EJS tags. See if you can find some docs around making it so VSCode supports EJS tag indentation.